pylint/pyreverse/writer.py

```diff
@@ -123,7 +153,41 @@ def get_title(self, obj):
         """get project title"""
         return obj.title
 
-    def get_values(self, obj):
+    def get_style(self):
```

Comment:
```suggestion
    def get_style(self) -> str:
```
Could you add typing on new functions please ? :)

Reply:
Will do!